cmd/influxd/run.go

```diff
@@ -176,7 +176,7 @@ func (cmd *RunCommand) Open(config *Config, join string) (*messaging.Broker, *in
 	// have it occasionally tell a data node in the cluster to run continuous queries
```

Comment:
I'm not sure I follow you.  This code gets called from the continuous query loop.  It also needs to know if it's the leader when it kicks off.